### ISSUE TITLE
Convert FixCollection to an instance class

### ISSUE THREAD

**Issue by n8rzz** [n8rzz/atc#116]
This will better solve the problem encountered with #115, and will remove the need to reset the collection every time an airport changes.

**Comment by n8rzz** [comment]
blocked by #77

**Comment by n8rzz** [comment]
this is not simply implemented at this time and it seems like _way_ more work, and added complexity, to get this working.  This may need to be revisited later but for now, out of scope.
